pyscriptjs/src/interpreter_client.ts

```diff
@@ -22,12 +23,17 @@ export class InterpreterClient extends Object {
     stdio: Stdio;
 
     constructor(
+        useWorker: boolean,
         config: AppConfig,
         stdio: Stdio,
         remote: Synclink.Remote<RemoteInterpreter>,
         unwrapped_remote: RemoteInterpreter,
     ) {
+        if (useWorker && unwrapped_remote !== undefined) {
+            throw new Error('AssertionError: cannot pass an unwrapped_remote ' + 'if useWorker === true');
+        }
```

Comment:
Instead of setting `unwrapped_remote` to undefined, we could make it into a `Proxy` that raises an internal error if we touch it.

Reply:
maybe, but hopefully `unwrapped_remote` will be eventually killed, so probably there is no point in over-engineering this